mods/ra/rules/misc.yaml

```diff
@@ -44,6 +44,7 @@ CRATE:
 		MaxAmount: 5
 		MinAmount: 1
 		MaxDuplicateValue: 1500
+		ValidTargets: GroundActor, WaterActor
```

Comment:
Actually this should also include aircraft. Currently only hind works, it probably has wrong target types though